sw/device/silicon_creator/mask_rom/BUILD

```diff
@@ -274,3 +274,23 @@ opentitan_functest(
         ],
     ),
 )
+
+# TODO: Add verilator test
+opentitan_functest(
+    name = "e2e_bootup_no_rom_ext_signature",
+    srcs = [
+        "mask_rom_test.c",
+    ],
+    cw310 = cw310_params(
+        args = [
+            "--exec=\"console -q -t0\"",
+            "--exec=\"bootstrap $(location {test_bin})\"",
+            "console",
+            "--timeout=3600",
+            "--exit-success='Bootstrap: DONE![\\s]*BFV_[0-9a-z]{{8}}\nLCV_[0-9a-z]{{8}}\n'",
```

Comment:
maybe it might be useful to declare a [shared variable](https://bazel.build/rules/tutorial-sharing-variables) in the rules/opentitan.bzl file for this regex? That way it is easier to keep this in sync with any mask ROM changes. @cfrantz wdyt?

Reply:
I thought a bit more about this, and I currently added it as a local variable just above the E2E tests since its use is probably pretty limited to these tests right now.